script/Arcueid/Day 7/07_00_ARC07_1D.txt

```diff
@@ -29,7 +29,7 @@ I was so caught up in all the dark thoughts swirling around my head that I've on
 [sha:a20d8ef0533decc2bfc9b21deb0c8e5523f5455d]{
 -- Page 113, Offset 16856.
 -- それがどういう意味なのか、本来なら考えるまでもない。
-Normally, there would be no need to think about the meaning behind those words.
+By all rights, those words shouldn't even warrant thought.
```

Comment:
This is another edit that turned out a little awkward, at the risk of deviating too far from the origin, may I suggest 'By all rights, those words shouldn't even be given a second thought.'

Reply:
I'm ok with "second thought" despite deviation from the literal Japanese because "does not warrant thought [beyond basic processin]" is basically what that says. With that said, I don't like the new structure of your suggestion. I'd propose:
> By all rights, those words shouldn't even warrant a second thought.